system-addon/content-src/components/Card/_Card.scss

```diff
@@ -190,56 +168,144 @@
     text-overflow: ellipsis;
     white-space: nowrap;
   }
+}
 
-  // Wide layout styles
-  @media (min-width: $break-point-widest) {
-    $line-height: 23px;
-    height: $card-height-large;
+.normal-cards {
+  .card-outer {
+    // Wide layout styles
+    @media (min-width: $break-point-widest) {
+      $line-height: 23px;
+      height: $card-height-large;
 
-    .card-preview-image-outer {
-      height: $card-preview-image-height-large;
+      .card-preview-image-outer {
+        height: $card-preview-image-height-large;
+      }
+
+      .card-details {
+        padding: 13px 16px 12px;
+      }
+
+      .card-text {
+        max-height: 6 * $line-height + $card-title-margin;
+      }
+
+      .card-host-name {
+        font-size: 12px;
+        padding-bottom: 5px;
+      }
+
+      .card-title {
+        font-size: 17px;
+        line-height: $line-height;
+        margin-bottom: 0;
+      }
+
+      .card-text:not(.no-description) {
+        .card-title {
+          max-height: 3 * $line-height;
+        }
+      }
+
+      .card-description {
+        font-size: 15px;
+        line-height: $line-height;
+      }
+
+      .card-context {
+        bottom: 4px;
+        font-size: 14px;
+      }
     }
+  }
+}
 
-    .card-details {
-      padding: 13px 16px 12px;
+.compact-cards {
+  .card-outer {
+    height: $card-height-compact;
+
+    .card-preview-image-outer {
+      height: $card-preview-image-height-compact;
     }
 
-    .card-text {
-      max-height: 6 * $line-height + $card-title-margin;
+    .card-details {
+      padding: 12px 16px;
     }
 
     .card-host-name {
-      font-size: 12px;
-      padding-bottom: 5px;
+      line-height: 10px;
     }
 
-    .card-download-icon {
-      &.icon-download-folder {
-        height: $large-download-folder-icon-size;
-        width: $large-download-folder-icon-size;
+    .card-text {
+      .card-title,
+      &:not(.no-description) .card-title {
+        font-size: 10px;
```

Comment:
This looks particularly small, but I suppose if that's what design wants… Even the site name for top sites under the images are 11px and those are usually a single word to glance at instead of several words to try to read. Maybe because next to Pocket titles, which are 17px, these titles are more than 40% smaller.

Reply:
Yeah, it is pretty small 🤔 . This might be one of those that gets tweaked once it gets in front of more eyeballs 👀